packages/client/src/pages/PortalSettings/categories/data-management/backup/manual-backup/index.js

```diff
@@ -298,6 +299,7 @@ class ManualBackup extends React.Component {
           {isCheckedTemporaryStorage && (
             <div className="manual-backup_buttons">
               <Button
+                id="create"
```

Comment:
Too simple id value (potential duplicates)

Reply:
Fix to "create-button"